config/prow/plugins.yaml

```diff
@@ -4,6 +4,7 @@ triggers:
 - repos:
   - kubernetes-sigs/cluster-api
   - kubernetes-sigs/cluster-api-provider-azure
+  - kubernetes-sigs/cluster-api-provider-vsphere
```

Comment:
Seems reasonable to add CAPV like core CAPI here. It should be for `trigger_github_workflows: true`